modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/menu.js

```diff
@@ -281,6 +283,8 @@ AUI.add(
 
 					menu = A.Node.create(TPL_MENU);
 
+					menu.addClass(CSS_SHOW);
```

Comment:
As clay css is including rules for the `show` selector, I'd like to double check this menu styling is not being affected by the new class we are adding. Have you seen any change in styling?

Is meny `<div>` getting more properties when show class is added?